src/Compiler/Service/service.fs

```diff
@@ -636,9 +638,6 @@ type BackgroundCompiler
         ) =
 
         node {
-            if useChangeNotifications then
-                do! builder.NotifyFileChanged(fileName, DateTime.UtcNow)
```

Comment:
If we're removing this we should probably update the doc string on `NotifyFileChanged` that says you don't have to call it if you check the changed file immediately.

Reply:
Yes, this was being spammed a lot in VS by various services.